modules/migrate/application/clicommands/NavigationCommand.php

```diff
@@ -31,165 +21,18 @@ class NavigationCommand extends Command
      */
-    public function indexAction()
-    {
-        $moduleManager = Icinga::app()->getModuleManager();
-        if (! $moduleManager->hasEnabled('icingadb')) {
-            Logger::error('Icinga DB module is not enabled. Please verify that the module is installed and enabled.');
-            return;
-        }
-
-        $preferencesPath = Config::resolvePath('preferences');
-        $sharedNavigation = Config::resolvePath('navigation');
-        if (! file_exists($preferencesPath) && ! file_exists($sharedNavigation)) {
-            Logger::info('There are no local user navigation items to migrate');
-            return;
-        }
-
-        $rc = 0;
-        $user = $this->params->get('user');
-        $directories = new DirectoryIterator($preferencesPath);
-
-        foreach ($directories as $directory) {
-            $username = $user;
-            if ($username !== null && $directories->key() !== $username) {
-                continue;
-            }
-
-            if ($username === null) {
-                $username = $directories->key();
-            }
-
-            $hostActions = $this->readFromIni($directory . '/host-actions.ini', $rc);
-            $serviceActions = $this->readFromIni($directory . '/service-actions.ini', $rc);
-
-            Logger::info('Migrating monitoring navigation items for user "%s" to the Icinga DB Web actions', $username);
-
-            if (! $hostActions->isEmpty()) {
-                $this->migrateNavigationItems($hostActions, $directory . '/icingadb-host-actions.ini', $rc);
-            }
-
-            if (! $serviceActions->isEmpty()) {
-                $this->migrateNavigationItems($serviceActions, $directory . '/icingadb-service-actions.ini', $rc);
-            }
-        }
-
-        // Start migrating shared navigation items
-        $hostActions = $this->readFromIni($sharedNavigation . '/host-actions.ini', $rc);
-        $serviceActions = $this->readFromIni($sharedNavigation . '/service-actions.ini', $rc);
-
-        Logger::info('Migrating shared monitoring navigation items to the Icinga DB Web actions');
-
-        if (! $hostActions->isEmpty()) {
-            $this->migrateNavigationItems($hostActions, $sharedNavigation . '/icingadb-host-actions.ini', $rc);
-        }
-
-        if (! $serviceActions->isEmpty()) {
-            $this->migrateNavigationItems($serviceActions, $sharedNavigation . '/icingadb-service-actions.ini', $rc);
-        }
-
-        if ($rc > 0) {
-            Logger::error('Failed to migrate some monitoring navigation items');
-            exit($rc);
-        }
-
-        Logger::info('Successfully migrated all local user monitoring navigation items');
-    }
-
-    /**
-     * Migrate the given config to the given new config path
-     *
-     * @param Config $config
-     * @param string $path
-     * @param int    $rc
-     */
-    private function migrateNavigationItems($config, $path, &$rc)
+    public function indexAction(): void
```

Comment:
This should still default to `--user *`, as described in the help. Though, only here, so backwards compatibility is still kept